src/transaction/log_append.hpp

```diff
@@ -74,6 +74,7 @@ struct log_append_info
 {
   int vdes;			/* Volume descriptor of active log */
   std::atomic<LOG_LSA> nxio_lsa;  /* Lowest log sequence number which has not been written to disk (for WAL). */
+  std::atomic<LOG_LSA> highest_evicted_lsa;
```

Comment:
A better place for highest_evicted_lsa is pgbuf_Pool. My suggestion is to move it there.

Also initialize it as NULL_LSA.